crates/revm/src/bsc/handler_register.rs

```diff
@@ -0,0 +1,65 @@
+//! Handler related to BNB-Smart-chain
+
+use crate::{
+    handler::register::EvmHandler,
+    interpreter::Gas,
+    primitives::{address, db::Database, spec_to_generic, Address, EVMError, Spec, SpecId, U256},
+    Context,
+};
+use std::sync::Arc;
+
+pub const SYSTEM_ADDRESS: Address = address!("fffffffffffffffffffffffffffffffffffffffe");
+
+pub fn bsc_handle_register<DB: Database, EXT>(handler: &mut EvmHandler<'_, EXT, DB>) {
+    spec_to_generic!(handler.cfg.spec_id, {
+        handler.post_execution.reward_beneficiary =
```

Comment:
if we dont assign other fields like `pre_execution`, will use default as the Ethereum network?

Reply:
Yes, it will overwrite the mannet setting
```
        let mut handler = Self::mainnet::<SPEC>();
        handler.cfg.is_bsc = true;
        handler.append_handler_register(HandleRegisters::Plain(
            crate::bsc::bsc_handle_register::<DB, EXT>,
        ));
```